classes/04class/exercises/c04-iac03/starkmatt/README.md

```diff
@@ -0,0 +1,14 @@
+# C04-IAC03
+
+## Command Execution Output
+- [output.txt](output.txt)
+
+## Questions:
+- Describe below why some changes just updated the resource being changed and the other required a full replacement of the resource.
+    - The VPC Cidr / Subnet Cidr changed without any issues because the change was made on the variable value itself, hence the data would be called the same way initally.
+    - The Subnet name couldn't because the variable name had changed, meaning that we needed to go back into the VPC.tf to change the variable name from ${var.private_subnet-b} to ${var.private_subnet-b1}  
```

Comment:
the exercise is not asking you to change the variable name but the VALUE of the variable

Reply:
I see, let me fix that up :D !